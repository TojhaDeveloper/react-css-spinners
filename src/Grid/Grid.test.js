import React from 'react'
import TestRenderer from 'react-test-renderer' // ES6
import { Grid } from '..'

it('renders correctly', () => {
  const tree = TestRenderer.create(<Grid />).toJSON()
  expect(tree).toMatchSnapshot()
})
