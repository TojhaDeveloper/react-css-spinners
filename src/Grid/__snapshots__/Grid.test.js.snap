// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <style>
     [object Object]
  </style>,
  <div
    className="lds-grid"
  >
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
    <div />
  </div>,
]
`;
